.lando.yml

```
@@ -32,7 +32,7 @@ services:
         - .:/usr/local/azqs-module-project
     build:
       # Create a new quickstart project and use the azqs_module_project module as a non-packagist repository.
-      - git clone --branch 2.2.x https://github.com/az-digital/az-quickstart-scaffolding.git /app
+      - git clone --branch 2.4.x https://github.com/az-digital/az-quickstart-scaffolding.git /app
```

Maybe it would be better to just use `main`?
```suggestion
      - git clone https://github.com/az-digital/az-quickstart-scaffolding.git /app
```